content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5763,25 +5763,14 @@ await prisma.user.findMany({
 
 ## Filter on non-unique fields with <inlinecode>UserWhereUniqueInput</inlinecode>
 
-From version 4.5.0, the generated type `UserWhereUniqueInput` on [`where`](#where) exposes all fields on the model, not just unique fields. To use this feature, [enable the `extendedWhereUnique` preview flag](#enable-the-ability-to-filter-on-non-unique-fields-with-userwhereuniqueinput).
+From version 5.0.0, the generated type `UserWhereUniqueInput` on [`where`](#where) exposes all fields on the model, not just unique fields. This feature is also available in versions 4.5.0 and later using the `extendedWhereUnique` preview flag.
```

Comment:
👍

Reply:
I could switch this to 
```suggestion
From version 5.0.0, the generated type `UserWhereUniqueInput` on [`where`](#where) exposes all fields on the model, not just unique fields. You can also enable this with the `extendedWhereUnique` Preview feature in versions `4.5.0` and later.
```

Reply:
Thsi is what GitHub shows me here: 
<img width="598" alt="image" src="https://github.com/prisma/docs/assets/183673/776485db-768e-44c1-8a53-591446cd98c3">
That change looks a bit weird.

What we usually would say is maybe "This was available under feature flag xyz between versions Y and Z" as it refers to a past state.